pkg/csi-handler/pv_checker.go

```diff
@@ -117,7 +117,9 @@ func (checker *PVHealthConditionChecker) CheckControllerListVolumeStatuses() err
 			checker.eventRecorder.Event(pvc, v1.EventTypeWarning, "VolumeConditionAbnormal", volumeCondition.GetMessage())
 		} else {
 			// Send recovery event if the abnormal event was sent and unexpired
-			return checker.sendRecoveryEventToPVC(pvc, volumeCondition)
+			if err := checker.sendRecoveryEventToPVC(pvc, volumeCondition); err != nil {
+				return err
```

Comment:
checker.sendRecoveryEventToPVC does not return an error.  Just change to the following would be good:
```
checker.sendRecoveryEventToPVC(pvc, volumeCondition)
```

Reply:
@fengzixu Also since checker.sendRecoveryEventToPVC always returns nil, is it necessary to have a return value?

Reply:
Yes. We don't need to return value here currently. Because we thought "sending recovery failed due to get event cache indexer" isn't critical error, retry it should be fine.